.pre-commit-config.yaml

```diff
@@ -28,12 +28,13 @@ repos:
         -   --target-version=py311
         -   --target-version=py312
         -   --force-exclude=dbt/adapters/events/adapter_types_pb2.py
+        additional_dependencies: [flaky]
 
 -   repo: https://github.com/pycqa/flake8
```

Comment:
I assume flake8 is smart enough to not hit sQL files which means we don't have to manually exclude them. But just making sure

Reply:
I believe that's the case. Either way, this is how it was configured before, so we would have hit that problem already.